crates/sui-graphql-rpc/schema/draft_schema.graphql

```diff
@@ -587,7 +587,14 @@ type Checkpoint {
   # Only available on the final checkpoint in an epoch
   endOfEpoch: EndOfEpochData
 
-  transactionConnection(first: Int, after: String, last: Int, before: String): TransactionBlockConnection
+  transactionBlockConnection(
+    first: Int,
+    after: String,
+    last: Int,
+    before: String,
+    # Enhancement (post-MVP) relies on compound filters.
+    filter: TransactionBlockFilter,
```

Comment:
So code should throw error when this is provided?

Reply:
We just won't add this parameter until after the MVP.